lnrpc/rpc.proto

```diff
@@ -1468,6 +1475,9 @@ message PendingChannelsResponse {
 
         int64 local_balance = 4 [ json_name = "local_balance" ];
         int64 remote_balance = 5 [ json_name = "remote_balance" ];
+       
+        int64 local_chan_reserve = 6 [json_name = "local_chan_reserve"];
```

Comment:
can also copy the new comments to these two fields. the others _should_ be populated, but we can at least start by making incremental progress

Reply:
Done.